share/commands/agent-check

```diff
@@ -7,4 +7,32 @@
 # @man - clean lock file if it is too big
 # @man - check and restore init files 
 # @man - check that promises have been properly copied
+# @man +
+# @man *Options*:
+# @man +
+# @man *-q*: run the agent in quiet mode (display only error messages)
+# @man +
+# @man *-c*: run the agent without color output
+
+. "${BASEDIR}/../lib/common.sh"
+
+QUIET=false
```

Comment:
Currently
* QUIET = less verbose than default
* VERBOSE = more verbose than default

I would prefer avoid using VERBOSE for this. I could be DISPLAY_OUTPUT, or something better.